src/file-upload-with-preview.js

```diff
@@ -101,6 +100,26 @@ export default class FileUploadWithPreview {
 
                 this.deleteFileAtIndex(selectedFileIndex)
             }
+
+            // on image click
+            if (event.target.matches('.custom-file-container__image-multi-preview')) {
+                // Get its token
+                let fileToken = event.target.querySelector('.custom-file-container__image-multi-preview__single-image-clear__icon').getAttribute('data-upload-token')
```

Comment:
The variables here can be initialized with `const` here instead of `let`. All good, I'll update it when I push out the new version.